### ISSUE TITLE
torch to onnx, affine_grid_generator operator is not supported

### ISSUE THREAD

**Issue by kadirbeytorun** [pytorch/pytorch#38222]
## 🐛 Bug

RuntimeError: Exporting the operator affine_grid_generator to ONNX opset version 11 is not supported. Please open a bug to request ONNX export support for the missing operator.

## To Reproduce

1. Just trying to convert this pytorch project in link to onnx model, using the code below in eval.py

"https://github.com/D-X-Y/landmark-detection/tree/master/SBR"

```
inputs = image.unsqueeze(0)
net.to(torch.device('cpu'))
input_names = [ "actual_input_1" ] + [ "learned_%d" % i for i in range(72) ]
output_names = [ "output1", "output2"]
torch.onnx.export(net,      # model being run
        inputs,                         # model input (or a tuple for multiple inputs)
        "sbr.onnx",                # where to save the model (can be a file or file-like object)
        verbose=True, input_names=input_names, output_names=output_names
       ,operator_export_type=torch.onnx.OperatorExportTypes.ONNX_ATEN_FALLBACK
        ,opset_version=12                  
        )
```

## Expected behavior

Obtaining onnx model file correctly

## Environment

PyTorch version: 1.6.0a0+f6b1c04
Is debug build: No
CUDA used to build PyTorch: 10.2

OS: Ubuntu 18.04.4 LTS
GCC version: (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0
CMake version: version 3.17.2

Python version: 3.6
Is CUDA available: Yes
CUDA runtime version: Could not collect
GPU models and configuration: GPU 0: GeForce GTX 1050 Ti
Nvidia driver version: 440.64.00
cuDNN version: /usr/lib/x86_64-linux-gnu/libcudnn.so.7.6.5

Versions of relevant libraries:
[pip] numpy==1.18.4
[pip] torch==1.6.0a0+f6b1c04
[conda] blas                      1.0                         mkl  
[conda] libblas                   3.8.0                    15_mkl    conda-forge
[conda] libcblas                  3.8.0                    15_mkl    conda-forge
[conda] liblapack                 3.8.0                    15_mkl    conda-forge
[conda] magma-cuda102             2.5.2                         1    pytorch
[conda] mkl                       2019.0                   pypi_0    pypi
[conda] mkl-include               2019.0                   pypi_0    pypi
[conda] mkl-service               2.3.0            py36he904b0f_0  
[conda] mkl_fft                   1.0.15           py36ha843d7b_0  
[conda] mkl_random                1.1.0            py36hd6b4f25_0  
[conda] numpy                     1.18.2                   pypi_0    pypi
[conda] numpy-base                1.18.1           py36hde5b4d6_1  
[conda] torch                     1.6.0a0+f6b1c04           dev_0    <develop>

cc @houseroad @spandantiwari @lara-hdr @BowenBao @neginraoof

**Comment by pfeatherstone** [comment]
Any news on when this will be supported? It would be cool to run spatial transformers in ONNX.

**Comment by spandantiwari** [comment]
We can put it on the backlog, but I don't have an ETA for this. Please consider submitting a PR for adding support.

**Comment by luvwinnie** [comment]
any news on this?

**Comment by zgj-gutou** [comment]
+1

**Comment by ZiyuBao** [comment]
Any news? It is 2022

**Comment by cxmscb** [comment]
Any news? It is 2022

**Comment by xiejinsen** [comment]
Any news? It is 2022/3/25

**Comment by imneonizer** [comment]
+1

**Comment by LiuHaoChina** [comment]
Any news? It is 2022/4/27

**Comment by BogdanovKirill** [comment]
up

**Comment by buptweixin** [comment]
Any news?

**Comment by justinchuby** [comment]
Closed as a duplicate of #30563
